api/nodes/serializers.py

```diff
@@ -216,6 +216,11 @@ class NodeSerializer(JSONAPISerializer):
         related_meta={'count': 'get_contrib_count'},
     )
 
+    implicit_contributors = RelationshipField(
```

Comment:
It occurs to me that, since we are doing this in a way that I wouldn't want to encourage external users from using, we probably shouldn't have a publicly visible relationship. Can you wrap this in a composition so that it only shows if the app using it has admin scope? That should mean, practically, that it doesn't show in the BAPI or any apps that aren't ours.

Reply:
(After going a couple of rounds, we decided just to mark the API Response for this view as being experimental or otherwise untrustworthy rather than hiding the relationship)